nixos/modules/services/x11/urserver.nix

```diff
@@ -0,0 +1,72 @@
+# urserver service
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.urserver;
+in {
+
+  options.services.urserver = {
+
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable Unified Remote server.
+      '';
+    };
+
+    usedTCPPorts = mkOption {
+      type = types.listOf types.int;
+      default = [ 9510 9512 ];
+      description = ''
+        TCP ports that are used by urserver.
+        By default port 9510 is used for the web server.
+        By default port 9512 is used for WiFi connections.
+      '';
+    };
+
+    usedUDPPorts = mkOption {
+      type = types.listOf types.int;
+      default = [ 9511 9512 ];
+      description = ''
+        UDP ports that are used by urserver.
+        By default port 9511 is used for automatic server discovery.
+        By default port 9512 is used for fast connections.
+      '';
+    };
+
+  };
+
+  config = mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.urserver ];
+
+    systemd.services.urserver = {
+      description = ''
+        Server for Unified Remote: The one-and-only remote for your computer.
+      '';
+      wantedBy = [ "graphical.target" ];
+      after = [ "network.target" ];
+      serviceConfig = {
+        Type = "forking";
+        ExecStart = ''
```

Comment:
If this has the option to run in the foreground that is preferred.

Reply:
I will look into this. I was having issues getting it to work and ended up here, but this was probably not causing my original issue.

Reply:
It seems the forking type is necessary, if I run it without it, the process dies.
If you run it from the command line it allows for command line input, so it's surely intended to run forked with --daemon I would say.